button.md

```diff
@@ -28,22 +28,132 @@ If you need to add a Harvest Button after the script has loaded, see [Adding But
-| <pre>data-group</pre>     | A JSON object containing `id` and `name` properties representing the group in your application that this item belongs to. If your application does not have a higher-level group, this may be omitted. For example, a Trello Board: `{"id":"0FdAjinR","name":"Landing Pages Overhaul"}`
-| <pre>data-account</pre>   | A JSON object containing an `id` property — used only to populate `%ACCOUNT_ID%` in the `permalink` configuration setting. If you are not using `%ACCOUNT_ID%`, there is no need to set this attribute.
-| <pre>data-default</pre>   | A JSON object containing a `project_name` or `project_code` property — used to pre-select a suggested project by either its name or code. This has to be an exact match. For example: `{"project_code":"GOOG123"}` or `{"project_name":"Project 123 for Google"}`
-| <pre>data-permalink</pre> | A URL linking back to the item in your application this timer is related to. This will be displayed alongside your timer in Harvest.
+| <pre>data-item (required)</pre>       | Object. A JSON object containing `id` and `name` properties representing the item in your application this timer is related to. `id` will populate `%ITEM_ID%` in the `permalink` global configuration setting. `name` will be filled into the Notes field of the timer dialog.
+| <pre>data-group (optional)</pre>      | Object. A JSON object containing `id` and `name` properties representing the group in your application that this item belongs to. `id` will populate `%GROUP_ID%` in the `permalink` global configuration setting. `name` will allow for creating a Harvest Project of that name to track time to. If your application does not have a higher-level group, this may be omitted.
+| <pre>data-account (optional)</pre>    | Object. A JSON object containing an `id` property — used only to populate `%ACCOUNT_ID%` in the `permalink` global configuration setting. If you are not using `%ACCOUNT_ID%`, there is no need to set this attribute. The account ID is tied to a personal access token which can be created or found at https://id.getharvest.com/developers 
+| <pre>data-default (optional)</pre>    | Object. A JSON object containing a `project_name` or `project_code` property — used to pre-select a suggested project by either its name or code. This has to be an exact match.**
+| <pre>data-permalink (*required)</pre> | String. A URL linking back to the item in your application this timer is related to. This will be displayed alongside your timer in Harvest.
+
+`*` You must have either `permalink` in your global configuration settings, or `data-permalink` in your timer/s attributes. Use the former if you only want to add one timer element or if you want all timer elements to have the same permalink. Use the latter if you want to have multiple timer elements with unique permalinks.
```

Comment:
I think timer/s attributes is meant to be timer's attributes?

Reply:
Good call!